documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc

```diff
@@ -0,0 +1,28 @@
+= Fuse Tooling - What's New in 10.3
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.3.0.AM2
+:page-product_id: jbt_core
+:page-product_version: 4.5.3.AM2
+
+== Display Fuse version corresponding to Camel version proposed
+
+When creating a new project, the Camel version can be selected among a long list. Now, the Fuse version is also displayed to help choosing version corresponding to the production version that users are using/targeting.
```

Comment:
When you create a new project, you select the Camel version from a list. Now, the list of Camel versions includes the Fuse version to help you choose the version that corresponds to your production version.